src/hls-adapter.js

```diff
@@ -423,6 +423,13 @@ export default class HlsAdapter extends BaseMediaSourceAdapter {
     this._onAddTrack = this._onAddTrack.bind(this);
     this._eventManager.listen(this._videoElement, 'addtrack', this._onAddTrack);
     this._videoElement.textTracks.onaddtrack = this._onAddTrack;
+
+    this._subtitleTrackInitializedPromise = new Promise(resolve => {
+      this._hls.on([Hlsjs.Events.SUBTITLE_TRACK_LOADED], () => {
+        resolve();
+        this._hls.off([Hlsjs.Events.SUBTITLE_TRACK_LOADED]);
```

Comment:
need to removed it in `_removeBindings` as well.
better if it would be in `_adapterEventsBindings` so it will remove then, but it will be added in `_addBindings`

Reply:
I can add it to _adapterEventsBindings, but then we will need to track with a counter or something if it's the first time the event is called or not